portal/migrations/versions/426c876ed746_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 426c876ed746
+Revises: b92dc277c384
+Create Date: 2017-05-12 11:57:29.387852
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '426c876ed746'
+down_revision = 'b92dc277c384'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('organizations', sa.Column('default_locale_id', sa.Integer(), nullable=True))
```

Comment:
@pbugni we already have the 'locales' relation (via organization_locales table). The 'default_locale_id' is that organization's default (or "highest priority") locale.

Basically, per Justin's story, when a user is first created and assigned to an org, we want to use that org's locale(s) to set the user's initial locale. But because there may be cases where an org has multiple locales assigned to it, the default_locale_id should dictate which specific locale should be assigned to that user.